R/location.R

```diff
@@ -132,6 +116,44 @@ orderly_location_add <- function(name, type, args, root = NULL, locate = TRUE) {
 }
 
 
+##' @rdname orderly_location_add
+##'
+##' @param path The path to the other archive root. This should
+##'   generally be an absolute path, or the behaviour of outpack will
+##'   be unreliable.
```

Comment:
Might be worth cleaning up and defining the semantics of a relative path at some point (not in this PR). IMO it should be relative to the orderly root, and be resolved each time we use the location, not when it is first added.

Reply:
Are you thinking here again of people on the cluster where directories "move"?